subworkflows/local/utils_nfcore_methylseq_pipeline/main.nf

```diff
@@ -16,9 +16,6 @@ include { completionSummary         } from '../../nf-core/utils_nfcore_pipeline'
 include { imNotification            } from '../../nf-core/utils_nfcore_pipeline'
 include { UTILS_NEXTFLOW_PIPELINE   } from '../../nf-core/utils_nextflow_pipeline'
 include { UTILS_NFCORE_PIPELINE     } from '../../nf-core/utils_nfcore_pipeline'
-include { workflowCitation          } from '../../nf-core/utils_nfcore_pipeline'
```

Comment:
Why remove this?

Reply:
see latest maintainers minutes blog

Reply:
Got it: https://nf-co.re/blog/2024/maintainers-minutes-2024-11-29#removal-of-workflowcitation